src/main/cms/cms_menu_misc.c

```diff
@@ -124,10 +127,11 @@ static const OSD_Entry menuMiscEntries[]=
 {
     { "-- MISC --", OME_Label, NULL, NULL, 0 },
 
-    { "MIN THR",      OME_UINT16,  NULL,          &(OSD_UINT16_t){ &motorConfig_minthrottle,              1000, 2000, 1 },      REBOOT_REQUIRED },
-    { "DIGITAL IDLE", OME_UINT8,   NULL,          &(OSD_UINT8_t) { &motorConfig_digitalIdleOffsetValue,      0,  200, 1 },      REBOOT_REQUIRED },
-    { "DEBUG MODE",   OME_TAB,     NULL,          &(OSD_TAB_t) { &systemConfig_debug_mode, DEBUG_COUNT - 1, debugModeNames },      0 },
-    { "RC PREV",      OME_Submenu, cmsMenuChange, &cmsx_menuRcPreview, 0},
+    { "MIN THR",        OME_UINT16,  NULL,          &(OSD_UINT16_t){ &motorConfig_minthrottle,            1000, 2000, 1 },      REBOOT_REQUIRED },
+    { "DIGITAL IDLE",   OME_UINT8,   NULL,          &(OSD_UINT8_t) { &motorConfig_digitalIdleOffsetValue,    0,  200, 1 },      REBOOT_REQUIRED },
+    { "CAM ANGLE COMP", OME_UINT8,   NULL,          &(OSD_UINT8_t) { &rxConfig_fpvCamAngleDegrees,           0,   90, 1 },      0 },
```

Comment:
Should probably be named "FPV CAM ANGLE" to be consistent with the Configurator. Also the "compensation" only takes affect if a mode is configured and enabled.

Also, no reason to arbitrarily insert this above of the "DEBUG MODE" entry. The debug mode will be used much more often then the FPV cam angle setting (which is a rarely used setting).

Reply:
Thanks for the feedback. Just updated the PR.